WebServer/pages/auth.html

```diff
@@ -0,0 +1,14 @@
+{% extends 'base.html' %}
+{% block content %}
+<font size="+3" face="impact" color="#A0D7DD">
```

Comment:
Please take into account that some tags (and the `font` particularly) are obsolete. Read on what to use instead (block-level elements and styles).

Reply:
Okay, thank you for the information.